src/libaktualizr/primary/CMakeLists.txt

```diff
@@ -23,10 +23,7 @@ target_link_libraries(t_aktualizr virtual_secondary)
 if (BUILD_OSTREE)
     add_aktualizr_test(NAME aktualizr_fullostree SOURCES aktualizr_fullostree_test.cc PROJECT_WORKING_DIRECTORY ARGS $<TARGET_FILE:uptane-generator> ${PROJECT_BINARY_DIR}/ostree_repo)
     set_target_properties(t_aktualizr_fullostree PROPERTIES LINK_FLAGS -Wl,--export-dynamic)
-    add_dependencies(t_aktualizr_fullostree ostree_mock uptane-generator make_ostree_sysroot)
-    set_tests_properties(test_aktualizr_fullostree PROPERTIES
-        ENVIRONMENT LD_PRELOAD=$<TARGET_FILE:ostree_mock>
-        LABELS "noptest")
+    add_dependencies(t_aktualizr_fullostree uptane-generator make_ostree_sysroot)
```

Comment:
Why was the `noptest` label removed? And am I understanding correctly that are you able to remove the `ostree_mock` dependency because this test has it's own internal mock?

Reply:
The label has been removed by mistake, the goal was to remove dependency on ostree_mock.
BTW, what does this label mean ?

Yes, the `aktualizr_fullostree_test.cc` test can be self-sufficient in this regard and doesn't require preloading of any library.

Reply:
AFAIK the labels are only used for grouping tests. We use this in CI to specify tests to run or to exclude from each run.

Reply:
ptest are running on bitbaked images and indeed the preloading was too brittle to work there. So maybe the label can indeed be removed in this case.